leo_gz_worlds/worlds/leo_empty.sdf

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<sdf version="1.8">
+  <world name="leo_empty">
+    <physics name="1ms" type="ignored">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+      <dart>
+        <!-- Heightmaps behave better with the bullet collision detector -->
+        <collision_detector>bullet</collision_detector>
```

Comment:
Can this be passed through gz sim arguments instead of hardcoded in the world description?